substrate/frame/examples/src/lib.rs

```diff
@@ -38,3 +38,6 @@
 //!
 //! - [**`pallet-example-split`**](./split): A simple example of a FRAME pallet demonstrating the
 //!   ability to split sections across multiple files.
+//!
+//! - [**`pallet-example-storage-migrations`**](./storage-migrations): An example pallet
+//!   demonstrating best-practices for writing storage migrations.
```

Comment:
Can't find `frame-examples` anywhere 🤔  https://paritytech.github.io/polkadot-sdk/master/polkadot_service/index.html?search=frame_examples

Reply:
https://paritytech.github.io/polkadot-sdk/master/pallet_examples/index.html

Reply:
But, when you render the docs, all the links are broken. Can you double check they work please?